conf/config.json

```diff
@@ -49,7 +49,11 @@
                     "port": 8000,
                     "https": false,
                     "type": "scality_s3",
-                    "raftCount": 8
+                    "raftCount": 8,
```

Comment:
just a note, I think we don't need this anymore since we ingest by bucket
probably need one pr to do cleanup to remove unused vals/variables

Reply:
Ya I agree 👍 
`raftCount` is still used though w/ ProvisionDispatcher. I'd like to remove use of the ProvisionDispatcher though, but I want things working first before making this change